### PR TITLE
Output installed packages in all remaining CIs

### PR DESCRIPTION
**Pull request by Kaushik-Iyer**
## Motivation
When debugging CI fails, the installed package versions are important information. This PR makes it easy to check them.
This PR changes all of the remaining CIs

## Description of the changes
Add pip freeze in all remaining CIs

### CODE REVIEW

**Review by Alnusjaponica — COMMENTED** [review]
You should add `Output installed packages` to job `docktest` in `sphinx-build.yml`

File: .github/workflows/dockerimage.yml
Comment (Alnusjaponica) [inline .github/workflows/dockerimage.yml:69]: This command doesn't run in the container. I think replacing 
```
pip freeze --all
```
with
```
docker run "${HUB_TAG}" sh -c "pip freeze --all"
```
would be one of the possible way to see the dependency in the container.

**Comment by Kaushik-Iyer** [comment]
Made the changes

**Review by gen740 — CHANGES_REQUESTED** [review]
I realized there are unnecessary trailing spaces before/after changed lines. Could you remove these trailing spaces?

(A lot of yaml files already had these trailing spaces too ...)

**Comment by Kaushik-Iyer** [comment]
image

I can't find any such spaces in the workflows directory through that command

**Comment by gen740** [comment]
Hmmm, that's weird. Is your os Windows?

On my computer, that command tells me trailing spaces.

Screenshot 2023-02-21 at 10 49 55 PM

**Comment by Kaushik-Iyer** [comment]
image

In my linux I am getting a similar output. How do I remove these trailing spaces? Do these refer to the one extra blank line that I have added after my code?

**Comment by gen740** [comment]
Use `sed` command to replace all trailing spaces. `ls | xargs -n 1 sed -i "" -E "s/^ +$//"`

**Review by Alnusjaponica — COMMENTED** [review]
I think you need to add another `Output installed packages` in `performance-benchmarks-bayesmark.yml` for job `report`.

**Review by Alnusjaponica — COMMENTED** [review]
I think you need to add another `Output installed packages` to the job `report` in `performance-benchmarks-bayesmark.yml`. Currently, you only added `Output installed packages` to the job `benchmarks`.

**Comment by Kaushik-Iyer** [comment]
@Alnusjaponica I have made the changes

**Comment by Alnusjaponica** [comment]
@Kaushik-Iyer Thank you for your response. I confirmed the changes.

**Comment by gen740** [comment]
@Kaushik-Iyer a friendly reminder :)

**Comment by Kaushik-Iyer** [comment]
Oh I'm so sorry, I was waiting for a reply and I didn't notice the thumbs up react, I'll make the PR today

**Comment by Kaushik-Iyer** [comment]
I have pushed the changes

**Review by Alnusjaponica — APPROVED** [review]
LGTM

**Review by Alnusjaponica — APPROVED** [review]

**Review by gen740 — APPROVED** [review]
LGTM!

**Review by toshihikoyanase — APPROVED** [review]
Let me merge this PR since it got two approvals. Thank you for your contribution.

**Pull request merged**
